config.example.wikimedia.yaml

```diff
@@ -39,7 +39,7 @@ wikimedia.org: &wikimedia.org
     - path: projects/wikimedia.org.yaml
       options:
         <<: *default_options
-        pageviews:
+        metrics:
```

Comment:
Ok, but what I mean is, don't we have to change RB anyway because we're
adding a new endpoint for /metrics/unique-devices next to
/metrics/pageviews ?  So while we do that we might as well rename this?

On Tue, Mar 29, 2016 at 10:20 AM, Marko Obrovac <notifications@github.com>
wrote:

> In config.example.wikimedia.yaml
> <https://github.com/wikimedia/restbase/pull/578#discussion_r57730852>:
>
> > @@ -39,7 +39,7 @@ wikimedia.org: &wikimedia.org
> >      - path: projects/wikimedia.org.yaml
> >        options:
> >          <<: *default_options
> > -        pageviews:
> > +        metrics:
>
> No, that's not true @milimetric <https://github.com/milimetric> . All
> this does is it renames an internal options object's property name. I'm not
> saying we shouldn't do this, I'm saying going through a config change
> procedure just to change a name that is seen nowhere and affects nothing is
> futile and a waste of time. Once we have some other config changes to do to
> RB, I'd be more than happy to include this small change too.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/wikimedia/restbase/pull/578/files/87d6cb7f9349d6533a2654b6409e37ae43a393ba#r57730852>
>